solution/planning_solutions.go

```diff
@@ -195,22 +200,22 @@ func GenerateSolutions(context context.Context, timeMatcher matching.Matcher, re
 	return
 }
 
-//TravelPlansDeduplication removes travel plans contain places that are permutations of each other
-func TravelPlansDeduplication(travelPlans []PlanningSolution) []PlanningSolution {
-	duplicatedPlans := make(map[string]bool)
-	results := make([]PlanningSolution, 0)
-
-	for _, travelPlan := range travelPlans {
-		placeIds := make([]string, len(travelPlan.PlaceIDS))
-		copy(placeIds, travelPlan.PlaceIDS)
-		radix.Sort(placeIds)
-		jointPlanIds := strings.Join(placeIds, "_")
-		if _, exists := duplicatedPlans[jointPlanIds]; !exists {
-			results = append(results, travelPlan)
-			duplicatedPlans[jointPlanIds] = true
-		}
+// returns true if the new travel plan can be added to priority queue
+func plansDeduplication(plans map[string]bool, newPlan PlanningSolution) bool {
+	jointPlaceIDs := jointPlaceIDs(newPlan)
```

Comment:
consider a different output var name, instead of jointPlaceIDs, maybe something like `planID`?
so we can also avoid reusing the same name with that of the function

Reply:
Renamed to `jointPlaceIdsForPlan`.